### PR TITLE
[CloudMinds] 10362-Drop Sensor

### PR DESCRIPTION
**Pull request by Kenny-SZ**
This LWM2M Object includes Drop Sensor information.

### CODE REVIEW

File: 10362.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        </Item>
+		<Item ID="51">
+			<Name>Drop Detection</Name>
+			<Operations>R</Operations>
+			<MultipleInstances>Single</MultipleInstances>
+			<Mandatory>Optional</Mandatory>
+			<Type>Boolean</Type>
+			<RangeEnumeration/>
+			<Units/>
+			<Description><![CDATA[If Drop Detection is enabled or not,
```
Comment (akeranen) [inline 10362.xml:74]: Boolean values are "true" and "false"

File: 10362.xml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+		
+	  	<Item ID="100">
+			<Name>Drop Detection Control</Name>
+			<Operations>E</Operations>
+			<MultipleInstances>Single</MultipleInstances>
+			<Mandatory>Mandatory</Mandatory>
+			<Type/>
+			<RangeEnumeration/>
+			<Units/>
+			<Description><![CDATA[Enable/Disable Drop Detection: Parameter: 1:Enable, 0:Disable]]></Description>
```
Comment (akeranen) [inline 10362.xml:86]: Should this be type boolean with true/false?

**Review by akeranen — CHANGES_REQUESTED** [review]

File: 10362.xml (hunk 2 above)
Comment (Kenny-SZ) [reply 10362.xml:86]: Yes, changed. Thanks.

File: 10362.xml (hunk 1 above)
Comment (Kenny-SZ) [reply 10362.xml:74]: Currently true for enable, false for not.

File: 10362.xml (hunk 2 above)
Comment (seanmcilroy29) [reply 10362.xml:86]: Incorrect Type/Operation combination - A Resource, which supports the "Execute" operation, MUST have no associated Data Type (none) encoded as an empty value in Object DDF file

Comment (Kenny-SZ) [reply 10362.xml:86]: Removed, thanks.

**Comment by mkgillmore** [comment]
Group agrees to merge 1/9/2020

**Pull request merged**